### PR TITLE
Save job posts

### PR DESCRIPTION
**Pull request by julietrinh1**
- Implemented save posts button
- user can save job posts and unsave them 
- Added Saved Post page to display all the saved jobs

### CODE REVIEW

File: frontend/src/App.js
Comment (fatema-a) [inline frontend/src/App.js:1]: no problem here just adding the path

File: frontend/src/styles/JobPostings.css
Comment (fatema-a) [inline frontend/src/styles/JobPostings.css:1]: just styling on job posting added

File: frontend/src/components/Jobs/SavedJobs.js
Comment (fatema-a) [inline frontend/src/components/Jobs/SavedJobs.js:1]: adding the component and functionality of liking jobs no problem here

File: frontend/src/components/Jobs/JobPageForSeekers.js
Comment (LeonZhang7) [inline frontend/src/components/Jobs/JobPageForSeekers.js:1]: I see you made it possible for a job seeker to save job postings into their saved job posting tab, works great

**Review by LeonZhang7 — APPROVED** [review]
approved, merging

**Pull request merged**